src/snowflake/snowpark/modin/plugin/PANDAS_CHANGELOG.md

```diff
@@ -1,378 +0,0 @@
-## 1.17.0 (2024-05-21)
-
-### Bug Fixes
-- Fixed bug when creating a DataFrame with a dict of Series objects.
-- Fixed issues with concat with index and column alignment that impacted datasets loaded from file
```

Comment:
where do we put the change log for Snowpark pandas now?

Reply:
We put it in CHANGELOG under `Snowpark pandas API Updates`, see this PR: https://github.com/snowflakedb/snowpark-python/pull/1635/files